### PR TITLE
Add another test in 'css-text-decoration' for text-decoration propagation in tables

### PR DESCRIPTION
**Pull request by Ahmad-S792**

### CODE REVIEW

**Comment by Ahmad-S792** [comment]
First time contributing to WPT, it is based on original test from Blink in this [commit](https://src.chromium.org/viewvc/blink?view=revision&revision=199637). I am planning to fix it in WebKit so @nt1m suggested that I should add it to WPT so we can track it better. All credits to original author.

File: css/css-text-decor/reference/text-decoration-propagation-05-ref.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>CSS Test: text-decoration propagation - Tables</title>
+<link rel="author" title="Timothy Loh" href="mailto:timloh@chromium.org" />
+<link rel="help" href="https://www.w3.org/TR/css-text-decor-3/#line-decoration">
+<link rel="match" href="reference/text-decoration-propagation-05-ref.html">
```
Comment (nt1m) [inline css/css-text-decor/reference/text-decoration-propagation-05-ref.html]: No need to point to the ref in the reference file.

File: css/css-text-decor/reference/text-decoration-propagation-05-ref.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>CSS Test: text-decoration propagation - Tables</title>
+<link rel="author" title="Timothy Loh" href="mailto:timloh@chromium.org" />
+<link rel="help" href="https://www.w3.org/TR/css-text-decor-3/#line-decoration">
```
Comment (nt1m) [inline css/css-text-decor/reference/text-decoration-propagation-05-ref.html:4]: Typically these are not in the reference file.

**Review by nt1m — APPROVED** [review]
Looks good, thanks!

File: css/css-text-decor/reference/text-decoration-propagation-05-ref.html (hunk 1 above)
Comment (Ahmad-S792) [reply css/css-text-decor/reference/text-decoration-propagation-05-ref.html]: Just fixed it in next push.

File: css/css-text-decor/reference/text-decoration-propagation-05-ref.html (hunk 2 above)
Comment (Ahmad-S792) [reply css/css-text-decor/reference/text-decoration-propagation-05-ref.html:4]: Hmm.. I copied from -04 test file, should I remove them?

Comment (nt1m) [reply css/css-text-decor/reference/text-decoration-propagation-05-ref.html:4]: No strong opinions here, I typically don't include them, but it's possible they're present in some other reference files

Comment (Ahmad-S792) [reply css/css-text-decor/reference/text-decoration-propagation-05-ref.html]: Done.

**Pull request merged**